openerp/addons/base/migrations/7.0.1.3/pre-migration.py

```diff
@@ -128,6 +128,13 @@ def update_base_sql(cr):
             "ALTER TABLE ir_act_client INHERIT ir_actions")
 
 
+def is_module_installed(cr, module):
+    cr.execute(
+        "SELECT id FROM ir_module_module "
+        "WHERE name=%s and state='installed'", (module,))
```

Comment:
Yeah, although simple, we avoid to forget details like the 'to upgrade' state, or if the installed condition change avoid versions, we can abstract it with one call.